resource/supported.go

```diff
@@ -307,7 +313,44 @@ func (a *AWS) elbs() (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
-	return output.LoadBalancerDescriptions, nil
+	loadBalancerNames := make([]string, len(output.LoadBalancerDescriptions))
```

Comment:
Your PR looks good, but I will create some unit tests in `supported_test.go`, as we now have quite some logic here, and matching wrong tags with wrong ELBs would really cause some bad user experience. Sorry, I am little bit conservative here, because this tool deletes stuff :)